app/assets/javascripts/reactivation_warning.js

```diff
@@ -0,0 +1,9 @@
+$(document).on('turbolinks:load', function(){
+  $('input#passenger_active').change(function(){
+      if($(this).is(':checked')){
+        alert("You are about to temporarily reactivate this passenger. \
+              \n Please note that, unless an updated doctor's note is \
+              \n obtained, this passenger will deactivate again at 4:30am.");
```

Comment:
Steps:
1. Write a function that checks the date in the date field and returns true or false depending on whether it is in the past or not
2. Use the function in this anonymous function up here